geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -132,23 +132,8 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType
       if (ranges.head.isInstanceOf[Get]) {
         GetPlan(filter, table, ranges.asInstanceOf[Seq[Get]], hbaseFilters, toFeatures)
       } else {
-        // we want to ensure some parallelism in our batch scanning
-        // as not all scans will take the same amount of time, we want to have multiple per-thread
-        // since scans are executed by a thread pool, that should balance the work and keep all threads occupied
-        val scansPerThread = 3
-        val scans = ranges.asInstanceOf[Seq[Scan]]
-        val minScans = if (ds.config.queryThreads == 1) { 1 } else { ds.config.queryThreads * scansPerThread }
-        if (scans.length >= minScans) {
-          ScanPlan(filter, table, scans, hbaseFilters, toFeatures)
-        } else {
-          // split up the scans so that we get some parallelism
-          val multiplier = math.ceil(minScans.toDouble / scans.length).toInt
-          val splitScans = scans.flatMap { scan =>
-            val splits = IndexAdapter.splitRange(scan.getStartRow, scan.getStopRow, multiplier)
-            splits.map { case (start, stop) => new Scan(scan).setStartRow(start).setStopRow(stop) }
-          }
-          ScanPlan(filter, table, splitScans, hbaseFilters, toFeatures)
-        }
+        // TODO: Bigtable does not support MultiRowRangeFilter so conditionally set this
```

Comment:
The next release is in the next business day or three/four, so it might be better to hold this PR unless we can wire up a fix?